public/app/plugins/panel/geomap/editor/StyleRuleEditor.tsx

```diff
@@ -117,6 +124,16 @@ export const StyleRuleEditor: FC<StandardEditorProps<FeatureStyleConfig, any, an
         <InlineField label="Style" labelWidth={LABEL_WIDTH} className={styles.color}>
           <ColorPicker color={value?.fillColor} onChange={onChangeColor} />
         </InlineField>
+        <InlineField label="Opacity">
```

Comment:
Do we need special opacity?  is the opacity built into color enough?

Reply:
Good point - I forgot that alpha can be accessed through the color picker. 👍